### PR TITLE
fix continue

### PR DESCRIPTION
**Pull request by danhloimta**

### CODE REVIEW

File: app/Http/Controllers/User/ReviewBookController.php [hunk 1]
```diff
@@ -61,25 +61,29 @@ public function store(ReviewRequest $request, $slug)
...
-                    $info = [
-                        'send_id' => Auth::id(),
-                        'receive_id' => $owner->id,
-                        'target_type' => config('model.target_type.review'),
-                        'target_id' => $review->id,
-                        'viewed' => config('model.viewed.false'),
-                    ];
-                    $this->notification->store($info);
+            $isReview = $this->review->checkReview($request->all())->toArray();
+            if(empty($isReview)) {
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/ReviewBookController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/Http/Controllers/User/ReviewBookController.php</br>Expected 1 space(s) after IF keyword; 0 found

File: app/Http/Controllers/User/ReviewBookController.php [hunk 2]
```diff
@@ -61,25 +61,29 @@ public function store(ReviewRequest $request, $slug)
...
+                if (isset($data) && $data != null) {
+                    $this->book->updateStar($data, $id);
+                }
+                return redirect()->route('books.show', $slug);
             }
-            $data = $this->review->find($id);
-            if (isset($data) && $data != null) {
-                $this->book->updateStar($data, $id);
-            }
+            return back();
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/ReviewBookController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/User/ReviewBookController.php</br>Blank line found at end of control structure

File: app/Http/Controllers/User/ReviewBookController.php [hunk 3]
```diff
@@ -61,25 +61,31 @@ public function store(ReviewRequest $request, $slug)
...
+                if (isset($data) && $data != null) {
+                    $this->book->updateStar($data, $id);
+                }
+
+                return redirect()->route('books.show', $slug);
             }
 
-            return redirect()->route('books.show', $slug);
+            return back();
+
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/ReviewBookController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/User/ReviewBookController.php</br>Blank line found at end of control structure

File: resources/views/book/books.blade.php [hunk 4]
```diff
@@ -148,15 +148,22 @@
                                             <div class="product-details">
                                                 <div class="product-rating">
                                                     <ul>
-                                                        @for ($i = 0; $i < $book->avg_star; $i++)
-                                                            <li><a href="#"><i class="fa fa-star"></i></a></li>
+                                                        @for ($k = 1; $k < $book->avg_star; $k++)
```
Comment (BunnyPi04) [inline resources/views/book/books.blade.php]: ko đặt tên biến thế này nhé
đặt tên có nghĩa vào sau này đứa khác đọc mới hiểu đc chứ

File: resources/views/book/books.blade.php [hunk 5]
```diff
@@ -171,7 +178,8 @@
                 <div class="pagination-area mt-50">
                     <div class="list-page-2">
                         <p>
-                            {{ __('settings.showing')  }} {{($books->currentpage()-1) * $books->perpage()+1 }} {{ __('settings.to') }} {{ $books->total() }} items
+                            {{ __('settings.showing')  }}
+                            {{ ($books->currentpage()-1) * $books->perpage()+1 }} {{ __('settings.to') }} {{ ($books->currentpage()-1) * $books->perpage() + $books->count() }} {{ __('settings.of') }} {{ $books->total() }} {{ __('settings.items') }}
```
Comment (BunnyPi04) [inline resources/views/book/books.blade.php]: convention

**Pull request merged**